extensions/mssql/src/mssql.d.ts

```diff
@@ -1379,6 +1380,20 @@ declare module 'mssql' {
 			containmentTypes: string[];
 			recoveryModels: string[];
 		}
+
+		export interface ServerPropertiesInfo extends SqlObject {
+			language: string[];
+			memory: number;
```

Comment:
This should have a unit in the name - e.g. `memoryInMb`

(same for all other non-count numeric values)